frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -132,6 +134,18 @@ export const CreateInternalCluster = withHandlePromise<
             data-test-id="osd-dropdown"
           />
         </FormGroup>
+        <div className="co-m-pane__heading--baseline ceph-ocs-install__pane--margin">
```

Comment:
Create a common component and use it at both places.

Reply:
I've almost completed  with the changes. I'll wait for your PR to merge to avoid conflicts..